### PR TITLE
add cv

### PR DESCRIPTION
**Pull request by sergiomons**

### CODE REVIEW

File: staff/sergiomons/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Sergio Monge Sosa</h1>
+    <p>Full-Stack Developer(Bueno lo intentaré ser)</p>
+    <h2>Experiencia profesional</h2>
+    <ul>
+        <li><h3>Ingeniería Fran López</h3></li>
```
Comment (EzequielBurgosL) [inline staff/sergiomons/cv/index.html]: I think it is recommended, not to put headers inside a list tag (Change the font-size of the list tags later on the CSS)

File: staff/sergiomons/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <p>Gestión de documentación.</p>
+        <li><h3>Armada Española</h3></li>
+        <p>Pringar como un cerdo.</p>
```
Comment (EzequielBurgosL) [inline staff/sergiomons/cv/index.html]: hahah!

File: staff/sergiomons/cv/index.html (hunk 1 above)
Comment (sergiomons) [reply staff/sergiomons/cv/index.html]: I agree with you!! i,m gonna correct it!

**Comment by abelhernando** [comment]
Bien!!

File: staff/sergiomons/cv/index.html [hunk 3]
```diff
@@ -8,22 +8,21 @@
...
+    <h3>Full-Stack Developer(Bueno lo intentaré ser)</h3>
+    <h4>Barcelona</h4>
     <h2>Experiencia profesional</h2>
     <ul>
-        <li><h3>Ingeniería Fran López</h3></li>
+        <li>Ingeniería Fran López</li>
         <p>Gestión de documentación.</p>
-        <li><h3>Armada Española</h3></li>
+        <li>Armada Española</li>
         <p>Pringar como un cerdo.</p>
```
Comment (EzequielBurgosL) [inline staff/sergiomons/cv/index.html]: Awesome! changes were nicely applied!

**Pull request merged**